### PR TITLE
BE-59 Process documents raw_text into text

### PR DESCRIPTION
**Pull request by Scr0ll**

### CODE REVIEW

File: antiplag/models.py [hunk 1]
```diff
@@ -56,6 +56,7 @@ def create_and_process_text(cls, submission=None, text_raw=None, file=None):
 
         # asynchronously preprocess raw text and update the model
         document.text = document.process_raw_text()
+        print(document.text)
```
Comment (MCFreddie777) [inline antiplag/models.py:59]: Unnecessary print

File: nlp/text_preprocessing.py [hunk 2]
```diff
@@ -264,59 +264,3 @@ def lemmatize_func(word_tokens, language="sk"):
     return lemmas
 
 
-def part_of_speech_tagging_func(text):
```
Comment (MCFreddie777) [inline nlp/text_preprocessing.py:267]: Were this functions removed on purpose?

File: antiplag/models.py [hunk 3]
```diff
@@ -65,8 +66,9 @@ def process_file(self):
         return extract_text_from_file(self.file.path)
 
     def process_raw_text(self):
-        # Replace with actual process raw text method.
-        return 'text'
+        os.environ["w2n.lang"] = self.language
```
Comment (MCFreddie777) [inline antiplag/models.py:69]: What if there are more parallel text_processing functions running at the same time? Won't the latter overwrite the environment variable? Cannot we solve this using parameters?

**Review by MCFreddie777 — CHANGES_REQUESTED** [review]

File: antiplag/models.py (hunk 3 above)
Comment (OliverKanat) [reply antiplag/models.py:69]: Agreed, setting env variable does not seem as best approach.

File: nlp/text_preprocessing.py (hunk 2 above)
Comment (Scr0ll) [reply nlp/text_preprocessing.py:267]: yes. they have no use right now

File: antiplag/models.py (hunk 3 above)
Comment (Scr0ll) [reply antiplag/models.py:69]: downloaded module word2numberi18n use that by default. there are other options but none of them is great for us.
this is searching process of that module for language definition:
image

Comment (MCFreddie777) [reply antiplag/models.py:69]: This is so wrong - why doesn't it accept lang as a parameter?

Cannot we find any better alternative?

Comment (Scr0ll) [reply antiplag/models.py:69]: This is only module i found that works for any language

Comment (Scr0ll) [reply antiplag/models.py:69]: maybe i can modify this module. Add some function for setting language before call..?

Comment (Scr0ll) [reply antiplag/models.py:69]: i can change this function 
image
like this and add line
image

Comment (MCFreddie777) [reply antiplag/models.py:69]: Well, not sure but dependencies should not be changed - only if we forked it and created our own package.

Comment (Scr0ll) [reply antiplag/models.py:69]: okay so what we do with this?
there are no other modules that can work with slovak language

Comment (MCFreddie777) [reply antiplag/models.py:69]: Alright, for now, leave it like it is , just please add a commentary
`#TODO: Would not work in parallel`


In the close future: 

@OliverKanat What do you think of creating our own fork and package and refactoring the function to accept the language argument before checking all the other possibilities (env, sys lang etc.)

Or @Scr0ll , seems like the package is not dead and creating an issue with PR there would be beneficial.
https://github.com/bastie/w2ni18n

**Comment by MCFreddie777** [comment]
Please resolve the conflicts .

**Review by MCFreddie777 — APPROVED** [review]

File: antiplag/models.py (hunk 3 above)
Comment (OliverKanat) [reply antiplag/models.py:69]: I think even Java is better solution than this.

Comment (OliverKanat) [reply antiplag/models.py:69]: I think creating an issue with PR is good idea and we can skip processing words to numbers until it's resolved.

**Pull request merged**